sdks/python/setup.cfg

```diff
@@ -26,3 +26,21 @@ verbosity=2
 # fast_coders_test and typecoders_test.
 exclude=fast_coders_test|typecoders_test
 
+[tool:pytest]
```

Comment:
Is the change from unittest to pytest with these configurations basically a no-op?